src/literal.nit

```diff
@@ -132,7 +132,12 @@ redef class ACharExpr
 	# Is the expression returning a Code Point ?
 	fun is_code_point: Bool do return prefix == "u"
 
-	redef fun text do return n_char.text
+	redef fun text
+	do
+		# Check if the value of the literal are computed to return it in string format
+		if value != null then return value.to_s
+		return n_char.text
```

Comment:
I'm not sure it's correct. For a token, `text` is literal text in she source. eg `'a'` (a 3 char string) with quotes, or `'\n'` (a 4 char string), whereas value is the real represented single char as computed in `src/literal.nit`